backend/apps/users/api.py

```
@@ -0,0 +1,108 @@
+User = get_user_model()
+router = Router(tags=["User"])
+
+
+class SignupRequest(BaseModel):
+    email: EmailStr
+    password: str
+    
+    @field_validator('email')
+    def validate_usls_email(cls, v):
+        if not v.endswith('@usls.edu.ph'):
+            raise ValueError('Email must be a valid USLS email address (@usls.edu.ph)')
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)